src/core_plugins/timelion/public/webpackShims/jquery-flot.js

```diff
@@ -1,7 +1,16 @@
+const $ = require('jquery');
+
+if (window) window.jQuery = $;
```

Comment:
Actually, @spalger can you confirm my understanding of webpack shims. I'm pretty sure I'm right, but it's probably best to ask the author here and confirm...